### DISCUSSION TITLE
Error message: System.IO.IOException

### DISCUSSION THREAD

**Discussion by stephwon** [Tyrrrz/DiscordChatExporter#584] [Q&A]
I got an error message when I tried to extract my chat. I tried many ways: dividing them into smaller chunks, redownloading discord and extractor, etc. but I still get a error message.
How can I resolve this issue?
![System IO IOException Error Message](https://user-images.githubusercontent.com/64761042/118449489-10599780-b6a8-11eb-9e14-4dd5a4d23b99.PNG)

**Comment by Tyrrrz** [comment]
It looks like it fails to edit file date on one of the downloaded media files. Are there any permission restrictions on the path you selected?

**Reply by stephwon** [reply]
there are no permission restrictions.

**Reply by Tyrrrz** [reply]
Created #585

**Reply by stephwon** [reply]
I forgot to further mention this error occurred when I tried to extract the chat with "Download media" and without as well.

**Reply by Tyrrrz** [reply]
It really shouldn't happen with download media off

**Reply by stephwon** [reply]
yes you are correct, the error message shows up when I try to extract with "Download media" sorry for the confusion
